src/screen.cpp

```diff
@@ -111,9 +111,11 @@ static bool is_screen_name_escape_seq(const wchar_t *code, size_t *resulting_len
     return true;
 }
 
-/// iTerm2 escape codes: CSI followed by ], terminated by either BEL or escape + backslash.
-/// See https://code.google.com/p/iterm2/wiki/ProprietaryEscapeCodes.
-static bool is_iterm2_escape_seq(const wchar_t *code, size_t *resulting_length) {
+/// Operating System Command (OSC) escape codes, used by iTerm2 and others:
+/// ESC followed by ], terminated by either BEL or escape + backslash.
+/// See https://invisible-island.net/xterm/ctlseqs/ctlseqs.html
+/// and https://code.google.com/p/iterm2/wiki/ProprietaryEscapeCodes.
```

Comment:
I'd go ahead and also update the iTerm URL. It doesn't work.

But this does: https://iterm2.com/documentation-escape-codes.html

Reply:
Makes sense.  Done.